paasta_tools/cli/cmds/status.py

```diff
@@ -385,7 +385,8 @@ def print_flinkcluster_status(
             fmt = """      {job_name: <32.32} {state: <11} {job_id} {start_time}
         {dashboard_url}"""
         else:
-            fmt = "      {job_name: <32.32} {state: <11} {start_time}"
+            fmt = """      {job_name: <32.32} {state: <11} {start_time}
+        {dashboard_url}"""
```

Comment:
Sorry if it's a dumb question, I don't know the codebase well. Would this have the same format that we have when we pass in `-v`? We want to just link to the main page of the flink dashboard here, without the jobs part.

Reply:
see the update in the description

Reply:
👍